index.js

```diff
@@ -89,6 +91,9 @@ if (Platform.OS === 'ios') {
     DocumentDirectoryPath: AudioRecorderManager.NSDocumentDirectoryPath,
     LibraryDirectoryPath: AudioRecorderManager.NSLibraryDirectoryPath,
   };
+  AudioSource = {
```

Comment:
Why is this added here as an empty dictionary?

Reply:
I forgot to remove it after I added initialization at the top. My bad.